exercises/concept/interest-is-interesting/.meta/exemplar.rs

```diff
@@ -0,0 +1,35 @@
+pub fn interest_rate(balance: f64) -> f64 {
+    if balance < 0.0 {
+        return 3.213;
+    }
+
+    if balance < 1000.0 {
+        return 0.5;
+    }
+
+    if balance < 5000.0 {
+        return 1.621;
+    }
+
+    return 2.475;
```

Comment:
We can avoid explicit `return` entirely.

```suggestion
    if balance < 0.0 {
        3.213
    } else if balance < 1000.0 {
        0.5
    } else if balance < 5000.0 {
        1.621
    } else {
        2.475
    }
```